guacamole-jetty/src/main/java/dev/crafting/guacamole/WebsocketTunnel.java

```diff
@@ -23,6 +25,16 @@ protected GuacamoleTunnel createTunnel(Session session,
         config.setParameter("resize-method", "display-update");
         config.setParameter("enable-font-smoothing", "true");
         config.setParameter("force-lossless", "true");
+        // get other params from url querystring
+        String[] allowedParams = new String[] { "username", "password", "width", "height" };
```

Comment:
Please only allow "width" and "height", as "username" and "password" will be specified by server.

Reply:
Will update in a separate patch.